rpcs3/Emu/Cell/Modules/sceNpUtil.cpp

```diff
@@ -1,52 +1,155 @@
 #include "stdafx.h"
 #include "Emu/Cell/PPUModule.h"
+#include "Emu/Cell/lv2/sys_ppu_thread.h"
 #include "Emu/IdManager.h"
+#include "Emu/System.h"
 
 #include "sceNp.h"
 #include "sceNpUtil.h"
+#include "sysPrxForUser.h"
 
 LOG_CHANNEL(sceNpUtil);
 
-error_code sceNpUtilBandwidthTestInitStart(u32 prio, u64 stack)
+struct bandwidth_test final
+{
+	static const u32 id_base = 0xf0000000;
+	static const u32 id_step = 0x00000100;
+	static const u32 id_count = 1024;
```

Comment:
this should probably be count 1 and step 1 ?
what should be the base ?

Reply:
This should not be, you should opt for FXO instead of IDM.